pkg/sentry/fs/proc/meminfo.go

```diff
@@ -87,6 +87,11 @@ func (d *meminfoData) ReadSeqFileData(ctx context.Context, h seqfile.SeqHandle)
 	fmt.Fprintf(&buf, "AnonPages:      %8d kB\n", anon/1024)
 	fmt.Fprintf(&buf, "Mapped:         %8d kB\n", file/1024) // doesn't count mapped tmpfs, which we don't know
 	fmt.Fprintf(&buf, "Shmem:          %8d kB\n", snapshot.Tmpfs/1024)
+	fmt.Fprintf(&buf, "HugePages_Total:     100\n")
+	fmt.Fprintf(&buf, "HugePages_Free:      100\n")
```

Comment:
These are counts of static hugepages (allocated using e.g. mmap(MAP_HUGETLB)) and should be 0.